index.js

```diff
@@ -34,7 +34,11 @@ module.exports = {
     }
     this.app = app;
 
-    let options = extend(defaultOptions, app.options['ember-bootstrap']);
+    let options = extend({}, defaultOptions, app.options['ember-bootstrap']);
+    if (process.env['BOOTSTRAPVERSION']) {
```

Comment:
Would be possible somehow I think. But why would we need to do that? I would keep this feature really private, not meant to be used by users.